plugins/shovel.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: shovel
+spec:
+  homepage: https://github.com/dodopizza/kubectl-shovel
+  shortDescription: Gather diagnostics for .NET Core applications
+  version: v0.1.0
+  description: |
+    Plugin for gathering diagnostics from running in Kubernetes .NET Core
+    applications
```

Comment:
You could elaborate a bit how this plugin works. In particular, you seem to rely on well-known .NET tools, so it could make sense to mention these.

Reply:
More information added here